packages/eslint-plugin/tests/rules/no-empty-lifecycle-method.test.ts

```diff
@@ -134,7 +140,7 @@ ruleTester.run(RULE_NAME, rule, {
         @Component()
         class Test {
           ngAfterContentChecked() { }
-          ~~~~~~~~~~~~~~~~~~~~~
+          ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
What was the reason for this change in reporting style?

In other rules we tend to go for the method name I think (e.g. even in this same PR you have a mixture)

Reply:
Sorry for the confusion. Everything should be fine now.